src/content/docs/more-integrations/best-practices-integration.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Best practices for integration users
+tags:
+  - Best practices
+  - Integrations
+metaDescription: Here are some best practices for controlling how you expose your New Relic data when you're using an integration.
+freshnessValidatedDate: 2024-03-25
+---
+
+As a New Relic administrator, you must operate your New Relic organization with the principle of least privilege. By following this principle, you can reduce the risk of unauthorized access to sensitive data. This includes assigning permissions based on job roles and responsibilities. In a [New Relic organization](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure/), account-based access allows you to control what information is available to the users in your organization. This ensures that sensitive data remains secure and only authorized personnel can view it. 
```

Comment:
I think we may need a brief description of what "principle of least privilege" is here